.travis.yml

```diff
@@ -0,0 +1,30 @@
+sudo: false
+language: go
+
+go:
+  - "1.11.x"
+  - tip
+
+os:
+  - linux
+  - osx
+  - windows
+
+install:
+  - make setup
+
+jobs:
+  include:
+    - stage: Lint
+      script:
+        - diff <(goimports -d $(find . -type f -name '*.go' -not -path "./vendor/*")) <(echo -n "")
+        - make lint
+      go: "1.11.x"
+      os: osx
+
+script:
+  - make test
+  - make build
+
+allow_failures:
```

Comment:
[todo] add windows in allow_failures because  Windows builds are in early access stage.